### PR TITLE
Location dropdown modified

### LINKED ISSUE #450: Create New Location
# Feature Request
**As a** peoplemover user
**I would** like the location field to be more consistent with the rest of the fields in the form
**so that** I have a consistent experience when editing products

## Acceptance Criteria
**Given** I create a new location on the ‘Create New Product’ or "Edit Product" modal
**When** I click Location field
**Then** dropdown expands and show disabled color CTA for ‘+ create ‘’' tag’
**And** dropdown and location field should be highlighted to indicate that it’s being selected
**And** remove arrow down icon (plain text field box)
**And** once user starts typing on the location field, CTA turns into active color
**And** by hitting Enter should create a new location tag
**And** when created a new location tag, dropdown collapse and arrow down icon appears on the location tag field

## Design
https://www.figma.com/file/S4XKeovlQqeRk64k2qzhf8gL/PeopleMover?node-id=5354%3A364
Go to right

## Notes
We may have to break this story up but we want to be clear about how this interaction should play out so that engineers are informed when they find a way to re-create or re-build this field.

### PR DESCRIPTION
**Pull request by willot**
## Issue
Resolves #450 

## What was done
- [x] Modified behavior of location drop down

## How to test
1. Verify that the new behavior math what is expected. Not everything could be implemented due to limitation of library used

Feature URL: https://featurebranchui.apps.pd01i.edc1.cf.ford.com/
Feature Branch Deployment: https://jenkins-fordlabs.apps.pd01e.edc1.cf.ford.com/job/PeopleMover/job/Deploy%20PeopleMover%20FeatureBranch/

### CODE REVIEW

File: ui/src/Products/ProductForm.tsx [hunk 1]
```diff
@@ -57,6 +57,29 @@ export const customStyles: StylesConfig = {
         height: '22px',
         marginRight: '3px',
     }),
+    noOptionsMessage: (base, state) => {
+        console.log("base", base)
+        return {
+            ...base,
+            backgroundColor: 'transparent',
+            // color: props.data.__isNew__ ? '#5463B0' : '#403D3D',
```
Comment (michaelrygiel) [inline ui/src/Products/ProductForm.tsx:65]: Remove this line, it is commented out

File: ui/src/Products/ProductForm.tsx [hunk 2]
```diff
@@ -57,6 +57,29 @@ export const customStyles: StylesConfig = {
...
+            fontFamily: 'Helvetica, sans-serif',
+            fontSize: '12px',
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'space-between',
+            padding: '0px 17px 0 11px',
+            height: '30px',
+            margin: '3px 0px',
+            // @ts-ignore
```
Comment (michaelrygiel) [inline ui/src/Products/ProductForm.tsx:74]: is this necessary? ts-ignore is sometimes smelly

**Review by michaelrygiel — COMMENTED** [review]
Looks good overall, just check my comments

**Review by alexwojtala — DISMISSED** [review]
Looks good

File: ui/src/ReusableComponents/ReactSelectStyles.tsx [hunk 3]
```diff
@@ -258,18 +258,23 @@ export const filterByStyles = {
...
-        {props.selectProps.menuIsOpen
-            ?
-            <i className="fas fa-caret-up greyIcon" data-testid={props.selectProps.name}/> :
-            <i className="fas fa-caret-down greyIcon" data-testid={props.selectProps.name}/>}
+        {
+            props.options.length === 0 ?
+                <i style={{display: 'none'}}/>
+                :
+                (props.selectProps.menuIsOpen
+                    ?
```
Comment (egriff38ford) [inline ui/src/ReusableComponents/ReactSelectStyles.tsx:266]: Kinda weird ternary formatting?

**Review by smle225 — APPROVED** [review]

**Pull request merged**